active/dceu.md

```diff
@@ -10,9 +10,13 @@ Delegated responsibilities TBD.
 
 - Chair: TBD
 - Co-Chair: TBD
-- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
-
+- Board Liaison: Çağıl Uluşahin Sönmez  
+- Other members:
+    Becky Smith
+    Benjamin Balder Bach
+    Raphael Michel
+    Tobias Kunze
+                         
 ## Future membership
 
 Membership is open to former organizers of DjangoCon EUs. Contact TBD if this is you and you want to join. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
```

Comment:
```suggestion
Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
```

Does this make sense? I don't know if the board really needs to approve new members, but leaving it in place in case it's a general requirement in working groups.